.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: 🚀 Feature Request/Idea
 about: Suggest a feature or idea for this project
-labels: 'type: feature, status: needs triage'
+labels: 'feature request, needs triage'
```

Comment:
We should unify repo labels on each repo at some point

Reply:
yeah I updated default labels for new repos but will have to update existing repos manually.